base/src/test/java/io/spine/validate/FieldValidatorShould.java

```diff
@@ -0,0 +1,74 @@
+package io.spine.validate;
+
+import com.google.common.collect.ImmutableList;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.List;
+
+import static com.google.common.collect.ImmutableList.of;
+import static io.spine.test.Verify.assertNotEmpty;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * @author Dmytro Dashenkov
+ */
+public abstract class FieldValidatorShould<V> {
+
+    protected abstract FieldValidator<V> validatedRequiredRepeatedField(List<V> values);
```

Comment:
As per my previous PR review. Something does not feel right, as you are passing the list, but obtaining a validator for the list item. From the signature I don't understand, why `Class<V> value` is not enough as an argument.

From the implementation I understand there is a FieldValidator, which requires something to validate. And you pass the list of values to validate. But that's totally unclear from both the method name and the absence of Javadoc. Perhaps, should be both renamed and documented.